src/Components/Assets/AssetManage.tsx

```diff
@@ -271,7 +271,7 @@ const AssetManage = (props: AssetManageProps) => {
       <div className="flex grow-0 flex-col md:w-[200px]">
         <div className="flex-start flex items-center">
           <div className="w-8">
-            <CareIcon className={`care-l-${item.icon} fill-gray-700 text-lg`} />
+            <CareIcon className={`care-${item.icon} fill-gray-700 text-lg`} />
```

Comment:
Let's use the `icon` prop here since we are anyways refactoring it...

```suggestion
            <CareIcon className="fill-gray-700 text-lg" icon={item.icon} />
```